src/assets/text/modelingText.js

```diff
@@ -1,76 +1,91 @@
+                'flubber_id': 'RNN',
+                'bees_id': 'RNN'
+            }],
+        "Graph convolutional network (RGCN)":[
             {
-                'title': "Graph convolutional network (RGCN)",
                 'method': "You don’t need to be a hydrologist to know water flows downstream, and that means the water temperature in downstream segments is likely to be similar to that in upstream segments. The spatial connections between streams allows us to leverage information across the stream network.",
-                'flubber_id':'RGCN'
+                'flubber_id': 'RGCN',
+                'bees_id': 'RGCN'
             },
             {
-                'title': "",
                 'method': "The way we inform the model about space is by calculating the distance between stream segments and allowing the model to weight information based on distance. The method we used is called a graph convolutional network.",
-                'flubber_id':'RGCN_2'
+                'flubber_id': 'RGCN_2',
+                'bees_id': 'RGCN'
             },
             {
-                'title': "",
                 'method': "In the DRB, adding information about the spatial configuration of the river network improves predictions.",
-                'flubber_id':'RGCN_2'
-            }, 
+                'flubber_id': 'RGCN_2',
+                'bees_id': 'RGCN'
+            }],
+        "Deep learning (RGCN_ptrn)":[
             {
-                'title': "Deep learning (RGCN_ptrn)",
                 'method': "Beyond space and time, We know quite a bit about how and why streams gain or lose heat – we shouldn’t ignore this wealth of knowledge simply because we’re using a data-driven approach. These process are intuitive – warm, sunny days should warm streams, and cold, cloudy days should cool streams.",
-                'flubber_id':'RGCN_ptrn'
+                'flubber_id': 'RGCN_ptrn',
+                'bees_id': 'RGCN_ptrn'
             },
             {
-                'title': "",
                 'method': "The way we inform our model about thermodynamics is by using a thermodynamic model to make predictions for all places and times.These predictions are used as “real data” to train our neural network.This creates a data- driven model that is emulating the physics of the system.",
-                'flubber_id':''
+                'flubber_id': '',
+                'bees_id': 'RGCN_ptrn'
             },
             {
-                'title': "",
                 'method': "And the great news? Now that the deep learning model is capturing the physical system, it can do more with less. Even under highly data sparse conditions (< 1% of training data), the model is still able to make accurate predictions.",
-                'flubber_id':''
-            },
-        ]
+                'flubber_id': '',
+                'bees_id': 'RGCN_ptrn'
+            }],
+        "Predicting water temperature at a national scale": [
```

Comment:
I think Sam's text structure has the map placed at the end of chapter 2 instead, but we can easily rearrange later